db.json

```diff
@@ -11,7 +11,7 @@
         "kwargs": {
             "name": "{{name}}"
         },
-        "last_edit": "Tue Jan 31 14:23:28 2023",
+        "last_edit": "Wed Feb 22 15:42:20 2023",
```

Comment:
It looks like `happi repair` updated the last_edit date of every device

Reply:
Hmm, that's expected with `.save()` but perhaps we should have a flag to not update timestamps...

Reply:
Ahh that's interesting behavior!